test/test_transcribe_utterance.py

```diff
@@ -24,56 +24,42 @@ class TestTranscribeUtterance(unittest.TestCase):
     """TranscribeUtterance unit test stubs"""
 
     def setUp(self):
-        self.model = TranscribeUtterance(action_url="TEST_URL")
+        self.model = TranscribeUtterance(
+            action_url="",
+        )
     
     def test_action_url(self):
         """Test TranscribeUtterance.action_url"""
         self.model.action_url = "TEST_STRING"
         assert self.model.get("action_url") == "TEST_STRING"
-        
-         
 
-     
     def test_play_beep(self):
         """Test TranscribeUtterance.play_beep"""
         self.model.play_beep = False
         assert self.model.get("play_beep") == False
-        
-         
 
-     
     def test_record(self):
         """Test TranscribeUtterance.record"""
-        self.model.record = TranscribeUtteranceRecord(save_recording=False, max_length_sec=1, rcrd_termination_silence_time_ms=1)
-        assert isinstance(self.model.record, TranscribeUtteranceRecord)
```

Comment:
Why is this removed?

Reply:
I believe latest commit should add the test back in